aunt-leahs-app/src/pages/VolunteerCheckInPage.jsx

```diff
@@ -11,23 +11,28 @@ import Header from '../components/Header'
 import SuccessStatus from '../components/SuccessStatus';
 
 import { clearStateAction } from '../redux/rootReducer';
+
 import moment from 'moment';
 
+
 function VolunteerCheckInPage({ location, volunteer, clearStateAction }) {
 
-	const [duration, setDuration] = useState();
+  const [duration, setDuration] = useState("");
 	const [isFormSubmitted, setIsFormSubmitted] = useState(false);
 	const [isSubmissionSuccessful, setIsSubmissionSuccessful] = useState(false);
 
 	const postShift = async () => {
 		try {
-
+      if (!duration.target) {
+        alert("Please select a shift time");
+        throw new Error("No time selected");
```

Comment:
I think just the alert message is sufficient, I can't think of a reason why there should be a console error